src/sentry/incidents/logic.py

```diff
@@ -1853,7 +1852,6 @@ def get_slack_actions_with_async_lookups(
                     context={
                         "organization": organization,
                         "access": SystemAccess(),
-                        "user": user,
```

Comment:
this appears to be completely unused -- I couldn't find anything in the output that relied on it :shrug: